newVersion.py

```diff
@@ -232,7 +232,8 @@ def pr_command(args, dir):
     pr_dir(args, dir)
 
 def pr1_command(args, dirs):
-    pr_command(args, dirs[0])
+    create_pr(args, dirs[0])
```

Comment:
Any reason not to pass the array to create_pr and just iterate the dirs?

Reply:
I only want the first two dirs (out of 4)

Reply:
Even though there is presumably an easy way to do that in Python, I think that adding the loop is more work than simply calling the function twice here. Later steps will call it once at a time.